src/dstack/_internal/core/models/configurations.py

```diff
@@ -247,25 +250,30 @@ def convert_port(cls, v) -> PortMapping:
         return v
 
     @validator("replicas")
-    def convert_replicas(cls, v: Range[int]) -> Range[int]:
+    def convert_replicas(cls, v: Any) -> Range[int]:
+        if isinstance(v, str) and ".." in v:
```

Comment:
It's not perfect that we duplicate range parsing logic. It is already implemented in [`Range`](https://github.com/dstackai/dstack/blob/04a62ba630fd775c05bbdeb0f9a7519a155bde46/src/dstack/_internal/core/models/resources.py#L26)